test/update_files.sh

```diff
@@ -0,0 +1,295 @@
+#!/bin/sh
```

Comment:
i assume this is just a copy paste like with the other cases. so ignoring this file